op-batcher/batch_submitter.go

```diff
@@ -313,7 +313,14 @@ mainLoop:
 				l.ch = ch
 			}
 			prevID := l.lastSubmittedBlock
-			for i := l.lastSubmittedBlock.Number + 1; i <= syncStatus.UnsafeL2.Number; i++ {
+			maxBlocksPerChannel := uint64(100)
```

Comment:
Should we make this configurable? Or increase it? Blocks can be really small.

Reply:
It's a would be nice, but this PR was focused on fixing the short term issue & we're going to do a bunch of work on this in a short bit as well.